app/models/translation.rb

```diff
@@ -76,6 +71,21 @@ def manifest_translated_phrases
     @manifest_translated_phrases ||= download_translated_phrases("name_description.xml")
   end
 
+  def push_published_to_s3
+    return if is_published
```

Comment:
Did the meaning of that boolean reverse? The old routine had `return unless`.

Reply:
previously publishing was a side effect of changing the `is_published` flag to true using an `before_update` hook.

We have refactored the publishing logic to be directly triggered and once it succeeds we update the `is_published` flag to true

Reply:
We have always had the constraint that a specific translation can only be published once and is considered immutable after it is successfully published